### DISCUSSION TITLE
Is it safe to use model trained in 3.1.x in spacy 3.0.x?

### DISCUSSION THREAD

**Discussion by erotavlas** [explosion/spaCy#8835] [Help: Other Questions] [answered]
If I retrain my NER model using latest version of spacy 3.1.1, is it ok to load this model in a previous version of spacy (3.0.5)?

**Comment by polm** [comment, marked as answer]
I don't think we've tested that. I wouldn't recommend it and I'm not sure it would work at all (you might just get an error). 

At a minimum I would recommend the same approach as taking models from 3.0 to 3.1, which is to try them on your data and see if performance is maintained. 

Is there a particular reason you'd want to be able to do this?

**Reply by erotavlas** [reply]
no, I was just wondering because in 2.x I think the models were compatible across versions, like if I trained it on 2.4, I could still use it in with 2.3.

**Reply by polm** [reply]
Models are generally supposed to be compatible across patch releases (the third number in versions), but I don't think we've ever tested backwards or assumed models would be used with earlier versions. I would be surprised if models were compatible with earlier minor versions (second number in versions), but again, never tested it.
